Tests/SentryTests/TestLogOutput.swift

```diff
@@ -1,9 +1,35 @@
 import Foundation
 
 class TestLogOutput: SentryLogOutput {
-    var loggedMessages: [String] = []
+    
+    private let queue = DispatchQueue(label: "TestLogOutput", attributes: .concurrent)
+    
+    private var _loggedMessages: [String] = []
+    
+    var loggedMessages: [String] {
+        get {
+            queue.sync {
+                return _loggedMessages
+            }
+        }
+    }
+    
     override func log(_ message: String) {
         super.log(message)
-        loggedMessages.append(message)
+        queue.async(flags: .barrier) {
+            self._loggedMessages.append(message)
+        }
+    }
+}
+
+class TestLogOutPutTests: XCTestCase {
+    
+    func testLoggingFromMulitpleThreads() {
+        let sut = TestLogOutput()
+        testConcurrentModifications(writeWork: { i in
+            sut.log("Some message \(i)")
```

Comment:
m: This will add too much unnecessary information in the actual log we use to debug CI.
Can we do something about it?

Reply:
`sut` is an extra instance of the `TestLogOutput`. It isn't a global one. All the other tests making use of it use their own instance, see https://github.com/search?q=repo%3Agetsentry%2Fsentry-cocoa%20%3D%20TestLogOutput()&type=code.

Reply:
I check the CI log and is really not there, so we're good to go.

But it does write to the log, and running locally I can see it in the console.
```
super.log(message)
```

I dont understand why this is not in the CI log